node/src/components/network/conman.rs

```diff
@@ -242,6 +247,66 @@ pub(crate) struct ProtocolHandshakeOutcome {
     pub(crate) handshake_outcome: HandshakeOutcome,
 }
 
+/// An error communicated back to a peer.
+#[derive(Debug, Deserialize, Error, Serialize)]
+enum PeerError {
+    /// The peer told us we are banned.
+    #[error("you are blocked")]
```

Comment:
```suggestion
    #[error("you are banned by a peer")]
```
I would have thought the `time_left` and `justification` should be in the display impl too, but I'll leave that up to you.

Reply:
No, good point. However, while `conman` uses "ban" internally (which is the preferred term), I sort of defined a "term boundary"; the rest of the node still uses "blocklist". This being public-facing, would mean it should be "blocked" - OTOH it is clearly part of conman, so I guess we can go with the term "ban" here :)

Reply:
Fixed in dea7b0b97